yammbs/_store.py

```diff
@@ -305,6 +306,37 @@ def get_mm_conformer_by_qcarchive_id(self, id: int, force_field: str):
                 .all()
             )
 
+    def get_qm_molecule_by_qcarchive_id(self, id: int) -> Molecule:
+        with self._get_session() as db:
+            conformer, smiles = next(
+                (conformer, smiles)
+                for (conformer, smiles) in db.db.query(
+                    DBQMConformerRecord.coordinates,
+                    DBQMConformerRecord.mapped_smiles,
+                )
+                .filter_by(qcarchive_id=id)
+                .all()
+            )
+            ret = Molecule.from_mapped_smiles(smiles, allow_undefined_stereo=True)
```

Comment:
I'm a little surprised by this (and the same below) - I thought the stereochemistry was stored in the SMILES and important?

Reply:
Oh that makes sense. I think I habitually/defensively add `allow_undefined_stereo` now, but if that's already been handled on loading then it makes sense not to include it here.

Reply:
I'm asking because I'm curious, not because I'm sure it's wrong. We'll go with whatever you/you three think is best, I just want to understand what's going on

Moved to here, though: https://github.com/openforcefield/yammbs/pull/70/files#diff-c6ae638f864fa58000d083968cc3b7c0b545e469c9b33e5f518a17f5976cb420R38

Reply:
Oh gotcha, I guess @amcisaac would know best since I haven't personally used yammbs in this way. Lexie, have you needed `allow_undefined_stereo` when building your `Molecule`s from the SMILES in the database? (More generally, is this basically the code you've written? I just guessed the shortest path of `Molecule.from_mapped_smiles` and `Molecule.add_conformer`.)

Reply:
Yes, this is how I do it, and (rarely) I do need `allow_undefined_stereo`. It's true that the stereochemistry should be included and is important, but I believe a few of our molecules have issues with stereochemistry and I was told by @j-wags and @lilyminium when I started that they usually add this flag to get around it, as it isn't so important that the molecules need to be excluded. I actually don't really know where those molecules come from or whether we want to include them in our benchmarks, however that's just how we've been doing it per inherited code. 

I'd argue in favor of at least allowing it as a user-passed flag, if we don't want to include it by default.

Reply:
For simplicity, and because this has been how things were already operating, I'm in favor of leaving this as-is. We can always change this in the future